portal-api/src/v1/controllers/utilisation-report-service/fee-record-corrections/fee-record-correction.controller.ts

```diff
@@ -50,3 +50,27 @@ export const getFeeRecordCorrection = async (req: GetFeeRecordCorrectionRequest,
     return res.status(errorStatus).send(errorMessage);
   }
 };
+
+/**
+ * Calls the DTFS Central API to submit a fee record correction.
+ * @param req - The request object containing information about the HTTP request.
+ * @param res - The response object used to send the HTTP response.
+ */
+export const saveFeeRecordCorrection = async (req: FeeRecordCorrectionRequest, res: Response) => {
+  try {
+    const { bankId, correctionId } = req.params;
+    const userId = req.user._id;
+
+    const data = await api.saveFeeRecordCorrection(bankId, Number(correctionId), userId);
+
+    return res.status(HttpStatusCode.Ok).send(data);
+  } catch (error) {
+    const errorMessage = 'Failed to save fee record correction';
+
+    console.error(errorMessage, error);
```

Comment:
%s %o

Reply:
Updated. I thought you had fixed all these instances generally across the codebase but there seems to still be lots?

Reply:
Oh I got confused it was other tech debt tickets that you did, that one is still in the backlog for in the future

Reply:
yep, I will be doing that one soon